### DISCUSSION TITLE
Can we get Whisper powered by Apple Neural Engine! 🔥

### DISCUSSION THREAD

**Discussion by holanino** [Marksdo/Whisper#20] [Ideas]
Can we get Whisper powered by Apple Neural Engine! 🔥

> All code is MIT-licensed.
> Upto 3x faster than the competition.
> Neural Engine as well as Metal runners.
> Open source CoreML models.
> 2 lines of code :)
> Whisper & Whisper-Turbo (even faster variant)

https://www.takeargmax.com/blog/whisperkit
https://github.com/argmaxinc/WhisperKit

**Comment by Marksdo** [comment]
Thank you for providing the relevant project materials. 
We will conduct research, and the current main branch of the project is addressing the requirement for speaker differentiation.

Current Whisper Mate will use Metal runners when running on Apple Silicon devices.
